docs/source/quickstart.rst

```diff
@@ -0,0 +1,325 @@
+Quickstart
+==========
+
+In this tutorial, we’ll assume that both Scrapy and Scrapy Streaming is already installed on your system.
+If that’s not the case, see :ref:`installation`.
+
+We'll develop the spider using Python because its simplicity, however, you can implement it using any programming language
+using the necessary syntax modifications.
+
+In this tutorial, we'll implement a simple spider to get the number of stars, open pull requests, and open issues in
+`scrapy-plugin's <https://github.com/scrapy-plugins>`_ repositories.
+
+Spider Behavior
+---------------
+
+If you are not familiar with Scrapy, we name Spider as an object that defines how scrapy should scrape
+information from a domain (or a group of domains). It contains the all the logic and necessary information to
+extract the data from a website.
+
+We'll define a simple spider, that works as follows:
```

Comment:
please specify what you are creating here (with a title like **Example 1: Github Streaming Spider**) also not sure if it is a good idea to define a github crawler, a site I would recommend for testing would be https://www.dmoz.org